src/App.tsx

```diff
@@ -1,14 +1,66 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+// import './App.scss';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { Todo } from './types/Todo';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { User } from './types/User';
+
+enum FilterBy {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[] | undefined>(undefined);
+  const [currentTodo, setCurrentTodo] = useState<Todo | null>(null);
+  const [user, setUser] = useState<User | null>(null);
+  const [query, setQuery] = useState('');
+  const [filterBy, setFilterBy] = useState('all');
+
+  useEffect(() => {
+    getTodos()
+      .then(data => setTodoList(data));
```

Comment:
can use short syntax
```suggestion
      .then(setTodoList);
```